src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -242,6 +248,8 @@ def __str__(self):
         storage_endpoint='core.windows.net',
         keyvault_dns='.vault.azure.net',
         sql_server_hostname='.database.windows.net',
+        mysql_server_hostname='.database.azure.com',
+        postgresql_server_hostname='.database.azure.com',
```

Comment:
Are they always the same ?

Reply:
yes for now, but I am not sure if they will change in the future.